bindings/go/constructor/default.go

```diff
@@ -0,0 +1,40 @@
+package constructor
+
+import (
+	"ocm.software/open-component-model/bindings/go/runtime"
+)
+
+var (
+	// Scheme is the default runtime scheme for the registry
+	Scheme = runtime.NewScheme()
+	// DefaultInputMethodRegistry is the default registry instance using the default scheme
+	DefaultInputMethodRegistry = New(Scheme)
+)
+
+// MustAddToScheme registers the file and UTF8 types with the given scheme.
+// It registers both versioned and unversioned types.
+func MustAddToScheme(scheme *runtime.Scheme) {
+	// scheme.MustRegisterWithAlias(&filev1.File{},
```

Comment:
Errr.. I assume this is for testing?

Reply:
it was a left over from my extraction to make the PRs smaller. I can remove it

Reply:
It's fine to leave it in. It is useful guide later. :)